common/src/main/java/com/microsoft/hydralab/common/entity/common/TestTask.java

```diff
@@ -164,6 +165,9 @@ public static TestTask convertToTestTask(TestTaskSpec testTaskSpec) {
         if (StringUtils.isNotBlank(testTaskSpec.runningType)) {
             testTask.setRunningType(testTaskSpec.runningType);
         }
+        if (StringUtils.isNotBlank(testTaskSpec.testPlan)) {
```

Comment:
You need to do the same operation in convertToTestTaskSpec method too. Hydra Lab will retry to trigger test task by convert TestTask to TestTaskSpec if meet exception when running.